data/starts.txt

```diff
@@ -0,0 +1,186 @@
+	choice
+		`	"Indeed."`
+		`	"How did you know?"`
+		`	"Sorry, not interested."`
+			decline
+	`	"I can see the will to explore in your eyes, and with your age it just seems like the perfect fit. I would be delighted to have you flying the 'Pristine Jewel'!" he says, taking you to the ship. It is clearly not brand new, but is still in a surprisingly good condition.`
+	scene "thumbnail/hai grasshopper"
+	`	"Because I like you I'll give you a friendly price. What do you say about... a bit over a million credits, for this wonderful ship? That's less than a third the price you would have to pay if you were to buy this model from the shipyard. Except you can't find a model like this in shipyards, trust me!"`
+	choice
+		`	"Sold!"`
+		`	"Sorry, not interested. I'll go to the shipyard instead."`
+			goto shipyard
+	`	As the two of you settle up the paperwork, the captain takes the time to explain, in great detail, why this ship is so marvelous. In your excitement to buy the ship all you can hear from the Hai captain's speech is the repeated reference to the Grasshopper's 'exceptional' reinforced armor.`
+	action
+		set "hai space start: Grasshopper"
+		give ship "Grasshopper (Expensive)" "Pristine Jewel"
+		payment -1132000
+	`	Before you leave, he gives you a final tip: "If you want to, you can always sell this ship and buy an Aphid instead. The shipyard authorities will give you a discount for having sold my ship."`
+		decline
+	label shipyard
+	`	The pictures and videos you've seen of Hai ships before simply don't do them justice; seeing them in all their splendor here in the shipyard is something else entirely. Even though you know a lot of those ships must have been in service for many years, you are stunned by how new and shiny they look.`
+
+
+# this ship uses the most expensive loadout possible (3.5M)
+# so that you +- have the same amount of value back if you sell it (1.3mill)
+# It has a small hull buff to help the player survive.
+ship "Grasshopper" "Grasshopper (Expensive)"
+	add attributes
+		"hull" 400
+	outfits
+		`"Benga" Atomic Thruster`
+		`"Biroo" Atomic Steering`
+		"Fuel Pod"
+		"Hai Chasm Batteries"
+		"Hai Corundum Regenerator"
+		"Hai Williwaw Cooling"
+		Hyperdrive
+		"Ion Cannon"
+		"Pebble Core"
+		"Pulse Rifle"
+		"Quantum Keystone" 2
+
+# the start itself cannot initiate events
+mission "Greenwater start"
+	landing
+	invisible
+	to offer
+		has "hai space start"
+	on offer
+		event "not from boston" 0
+		event "grew up on greenwater" 0
+		log "Finally scraped together enough money for a down payment on a starship. The interest on the mortgage is exorbitant."
+		log "Factions" "Republic" `Hundreds of years ago, the independent territories in different parts of human space agreed to join together into a single democratic government, with Earth as its capital. The rise of the Republic ushered in a long period of peace and prosperity in human history.`
```

Comment:
Does the player know enough about the Republic to make entries in their log? Maybe they were born to human parents in Hai space. 
It would be kind of cool to have a `First Contact: Republic`

Reply:
The planet you start on has a lot of humans on it, and the hai would definitely know about the republic

Reply:
Okay.